edxsearch/urls.py

```diff
@@ -8,4 +9,7 @@
 
 # urlpatterns is the standard name to use here
 # pylint: disable=invalid-name
-urlpatterns = patterns('', url(r'^search/', include(search.urls)),)
+if django.VERSION < (1, 9):
+    urlpatterns = [url(r'^search/', include(search.urls))]
```

Comment:
Not a blocker, but I do wonder which is preferred...forking and dealing with everything at the same time and making the difference clear or leaving deprecation warnings with no clear indication in code why they weren't yet handled.